validator/client/service.go

```diff
@@ -194,9 +194,15 @@ func (v *ValidatorService) Start() {
 		return
 	}
 
+	u := strings.ReplaceAll(v.conn.GetBeaconApiUrl(), " ", "")
+	hosts := strings.Split(u, ",")
+	if len(hosts) == 0 {
+		log.WithError(err).Error("No beacon API hosts provided")
```

Comment:
```suggestion
		log.WithError(err).Error("No API hosts provided")
```
This is better because we use custom endpoints too